### PR TITLE
adds iso pill packets, tweaks iso pills

### PR DESCRIPTION
**Pull request by 567Turtle**
## About The Pull Request

name, can be gotten from marine med, also iso pills are now slightly more efficient

## Why It's Good For The Game

twas no way for a joe shmoe to lay hand upon the panacea that are the isotonic pills (added by yours truly), plus it doesnt make the game stop working so

the isotonic mix was kind of mid before, found a way to make it better using the same stuff

## Changelog

:cl:
add: isotonic pill packets in the marinemed
balance: isotonic pills are now slightly more efficient
/:cl:

### CODE REVIEW

File: code/game/objects/items/storage/pill_packets.dm [hunk 1]
```diff
@@ -49,6 +49,14 @@
 	storage_slots = 8
 	max_storage_space = 8
 
+/obj/item/storage/pill_bottle/packet/isotonic
+	name = "Isotonic pill packet"
```
Comment (QualityVan) [inline code/game/objects/items/storage/pill_packets.dm:53]:
```suggestion
	name = "\improper Isotonic pill packet"
```

Comment (567Turtle) [reply code/game/objects/items/storage/pill_packets.dm:53]: Why `/improper`, none of the other pill packet names have it?

Comment (QualityVan) [reply code/game/objects/items/storage/pill_packets.dm:53]: It's for autoformating in sentences. Anything that isn't a proper noun and starts with a capital should have it.

**Comment by Naaanii** [comment]
None of the pill packets have a limit and isotonic injectors have a limit.

Please make the injectors infinite as well.

File: code/game/objects/items/reagent_containers/pill.dm [hunk 2]
```diff
@@ -159,8 +159,8 @@
 	pill_id = 7
 
 /obj/item/reagent_containers/pill/isotonic
-	pill_desc = "A pill with an isotonic solution inside. Used to stimulate blood regeneration."
-	list_reagents = list(/datum/reagent/consumable/nutriment = 6, /datum/reagent/iron = 6, /datum/reagent/consumable/sugar = 3)
+	pill_desc = "A pill with an isotonic solution inside. Used to stimulate blood regeneration. Do not take more then 4."
+	list_reagents = list(/datum/reagent/consumable/nutriment = 7.5, /datum/reagent/iron = 7.5)
```
Comment (TiviPlus) [inline code/game/objects/items/reagent_containers/pill.dm:163]: ?

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: code/game/objects/items/reagent_containers/pill.dm (hunk 2 above)
Comment (Xander3359) [reply code/game/objects/items/reagent_containers/pill.dm:163]: Stealth buff lmao

Comment (567Turtle) [reply code/game/objects/items/reagent_containers/pill.dm:163]: Thought i renamed the pr, forgot to, sorry. I realized (was yelled at by nanite users) that the current isotonic mixture is kind of bad, so I tweaked it just a bit to make it a little more efficient.

**Comment by 567Turtle** [comment]
> Please make the injectors infinite as well.

did and done

**Comment by Lumipharon** [comment]
Sorry I'll hold off until Xander's pr goes through

**Comment by InterroLouis** [comment]
You seem to have made it 1 instead of the -1 needed to make the injectors infinite.

**Comment by 567Turtle** [comment]
> You seem to have made it 1 instead of the -1 needed to make the injectors infinite.

I just went and checked, it says -1 in the code

**Comment by Xander3359** [comment]
You are going to need to update your packet because of this being merged https://github.com/tgstation/TerraGov-Marine-Corps/pull/12134.

**Pull request closed without merging**